corehq/apps/app_manager/suite_xml/sections/details.py

```diff
@@ -116,31 +116,23 @@ def get_section_elements(self):
                         )  # list of DetailColumnInfo named tuples
                     if detail_column_infos:
                         detail_id = id_strings.detail(module, detail_type)
-                        if detail.case_tile_template:
-                            helper = CaseTileHelper(self.app, module, detail, detail_id, detail_type,
-                                self.build_profile_id, detail_column_infos, self.entries_helper)
-
-                            d = helper.build_case_tile_detail()
-                            self._add_custom_variables(detail, d)
+                        print_template_path = None
+                        if detail.print_template:
+                            print_template_path = detail.print_template['path']
```

Comment:
good candidate to extract into a function to keep complexity down.

Reply:
extracted into a function, which was so tiny I decided to just inline it: 701e5cc2361